manifests/argo-rollouts.yaml

```diff
@@ -287,9 +334,29 @@ spec:
         app.kubernetes.io/name: argo-rollouts
     spec:
       containers:
-      - image: docker.io/argoproj/argo-rollouts:{{.argoRollouts.version}}
+      - image: quay.io/argoproj/argo-rollouts:latest
```

Comment:
Looks like the kustomize image needs updating to reflect quay.io so that the tag gets patched